### DISCUSSION TITLE
2017 年总结 - 写在转职后的一个月 · /2017-summary

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#519] [Announcements]
https://draveness.me/2017-summary

**Comment by zesming** [comment]
妈的谁都不服就服你，读了这么多书文章资料牛逼的不行，现在想来看些东西发现都没时间去做了，区块链的东西看了一点点就开始投机最后只学到了“哪家赌友天天输”，年初定的 todo list 毛都没做，灯塔果然不是一天建成的，不得不摩拜…ofo…小蓝…酷骑…

**Comment by zmecust** [comment]
我也是今年六月份毕业的，但你比我牛多了，向你学习。

**Comment by draveness** [comment]
> @zesming
> 妈的谁都不服就服你，读了这么多书文章资料牛逼的不行，现在想来看些东西发现都没时间去做了，区块链的东西看了一点点就开始投机最后只学到了“哪家赌友天天输”，年初定的 todo list 毛都没做，灯塔果然不是一天建成的，不得不摩拜…ofo…小蓝…酷骑…

你忘了 unibike 哈哈哈

**Comment by wujunze** [comment]
🐂

**Comment by authhwang** [comment]
博主有没有打算看go呀？

**Comment by draveness** [comment]
> @authhwang
> 博主有没有打算看go呀？

确实看了，而且还写了些小的服务

**Comment by er567** [comment]
同今年毕业，感觉自己还差的太远...
总是这里看看那里看看，然后发现还是都不会

**Comment by ximeng123** [comment]
感觉跟楼主差距好大

**Comment by craneyuan** [comment]
楼主厉害 :thumbsup:

**Comment by footmanFF** [comment]
博主的这个博客是用什么啥搭的啊 github pages 吗

**Comment by HorexC** [comment]
请问下是用什么软件来画图的呀(就MVC那个图)。感觉挺好的～

**Comment by craneyuan** [comment]
@HorexC 
![image](https://user-images.githubusercontent.com/8643491/35320267-2107fd3a-011e-11e8-932d-694e97a215b0.png)

**Comment by qiushi0908** [comment]
博主是研究生毕业吗？文章的深度都不错呀。我是最初研究AsyncDisplayKit看到你在segmentfault上的文章，google上检索了很多，只有你剖析的最精彩、细致，关注你后发现每篇文章都是精品，赞赞👍👍

**Comment by draveness** [comment]
> @qiushi0908
> 博主是研究生毕业吗？文章的深度都不错呀。我是最初研究AsyncDisplayKit看到你在segmentfault上的文章，google上检索了很多，只有你剖析的最精彩、细致，关注你后发现每篇文章都是精品，赞赞👍👍

刚本科毕业半年吧，谢谢

**Comment by ayayou** [comment]
还以为是个老司机，没想到刚毕业。。。我还能说什么呢。。。已经哭晕在厕所

**Comment by importpy** [comment]
在呢

**Comment by cizel** [comment]
比较好奇, 图片是用什么软件画的? 推荐一下呗

**Comment by shaveKevin** [comment]
比你牛b的人还这么努力，还有什么不努力的呢？
2018努力使自己变得更好！

**Comment by flynn-z** [comment]
博主 todolist是什么工具呀

**Comment by draveness** [comment]
> @PositiveEightZERO
> 博主 todolist是什么工具呀

wunderlist

**Comment by s2eker** [comment]
🐂

**Comment by doraeqing** [comment]
一直以你你是一个老司机。。。谁知道才毕业

**Comment by Cofecit** [comment]
测一下跨站攻击：

<script async src="http://cdn.bootcss.com/mathjax/2.7.0/MathJax.js?config=TeX-MML-AM_CHTML" integrity="sha384-crwIf/BuaWM9rM65iM+dWFldgQ1Un8jWZMuh3puxb8TOY9+linwLoI7ZHZT+aekW" crossorigin="anonymous"></script>

**Comment by draveness** [comment]
> @LimyronChin
> 测一下跨站攻击：
> 
> <script async src="http://cdn.bootcss.com/mathjax/2.7.0/MathJax.js?config=TeX-MML-AM_CHTML" integrity="sha384-crwIf/BuaWM9rM65iM+dWFldgQ1Un8jWZMuh3puxb8TOY9+linwLoI7ZHZT+aekW" crossorigin="anonymous"></script>

好使么[微笑]

**Comment by Jinxiansen** [comment]
叼那个叼！

**Comment by AAChartModel** [comment]
我靠,看到作者广博的知识面,旺盛的创作力,流畅的行文和精细的构图,我还以为大佬是工作了很多年的高龄猿类,结果你才是17年刚刚毕业,惊掉了下巴😲😲😲

**Comment by imerakk** [comment]
我在大学的时候也是天天吐槽java臃肿难用，现在一看都是java遍地都是。

**Comment by draveness** [comment]
> @coderLCX
> 我在大学的时候也是天天吐槽java臃肿难用，现在一看都是java遍地都是。

然而现在还是不写 Java 了

**Comment by AAChartModel** [comment]
所以 Java 到底为什么这么受欢迎呢?

**Comment by draveness** [comment]
> @AAChartModel
> 所以 Java 到底为什么这么受欢迎呢?

简单、不容易出错，好招人

**Comment by LJ8116** [comment]
大神竟然是17年毕业。。。我这10年真是白活了

**Comment by orangle** [comment]
坚持！加油！

**Comment by xinqiyang** [comment]
博主现在在那个城市，看积累不错，我们公司做区块链开发的，有兴趣可以聊聊，加我微信，xinqiyang

**Comment by AAChartModel** [comment]
这就是为什么你应该在开源社区如此活跃的原因,千里马常有而伯乐不常有.
博主的伯乐来了

**Comment by Sunnykaby** [comment]
厉害的不行啊。大神博客用什么写的。sketch都用的这么溜。不服不行

**Comment by Sunnykaby** [comment]
OK，找到了。

**Comment by kkliuxin** [comment]
大神 docker有推荐的书吗？

**Comment by qizidev** [comment]
楼主可以分享下总结出的排版和制图的规则么？

**Comment by niyaoyao** [comment]
男神你为何如此之 diao！

**Comment by AAChartModel** [comment]
我们还需要更多像您这样的天赋异禀又刻苦努力的程序猿🤣🤣🤣

**Comment by KiceBryant** [comment]
最近开始沉迷BLOCK CHIAN，看到你的BLOG风格又一次沉沦，前行者向你学习。

**Comment by MarkStray** [comment]
看第一篇文章【mysql-innodb】我以为是某行业十年资深老鸟，得知是学生刚毕业时，我想怎么着也得是研究生吧，万万没想到你才刚毕业

**Comment by Yarnus** [comment]
怎么人与人之间的差距就这么大呢 太厉害了 佩服 先收藏了

**Comment by AAChartModel** [comment]
我是通过iOS开发了解到楼主的,不过楼主貌似现在从iOS开发转业,去往了更加广阔的天地了😂😂😂

**Comment by qiushi0908** [comment]
@AAChartModel 去了阿里

**Comment by AAChartModel** [comment]
可以可以,大佬很会玩!!!

**Comment by draveness** [comment]
> @qiushi0908
> @AAChartModel 去了阿里

没去阿里

**Comment by AAChartModel** [comment]
在博主的评论区堂而皇之地发广告真的合适么?

**Comment by leoxiao2012** [comment]
啥都不说了，我爱你

**Comment by silencewwt** [comment]
> 在 7 月份至今的一段时间内，基本每天都在学习新的知识与技术

太可怕了, 向大佬学习!

**Comment by choleraehyq** [comment]
想问一下作者是哪司的？

**Comment by forezp** [comment]
我想认识认识博主，不知道是否有机会

**Comment by lidelin** [comment]
我比较好奇博主总结的 '排版和制图的规则' 是啥~
哈哈~~

**Comment by OC0311** [comment]
博主一年看那么多书是怎么做到的，挑知识点精读的吗？，我感觉我一年十本够呛了都

**Comment by draveness** [comment]
> 博主一年看那么多书是怎么做到的，挑知识点精读的吗？，我感觉我一年十本够呛了都

17 年上半年还没毕业，整块的时间比较多

**Comment by soulmz** [comment]
不说了，哭晕在厕所。

技术沉淀的这么深厚。

已订阅了你的博客。😂

**Comment by Cherrison** [comment]
大佬有读完CSAPP嘛 花费大量时间读完这个值吗, 现在有点纠结, 面试貌似也不会深入问这些.
花费那么多时间读CSAPP, 做它实验,,, 最后面试还不如那些, 刷1-200算法题的.
春招面试没刷算法题好伤啊

**Comment by draveness** [comment]
上了一次读已经好几年前了，最近又开始重新读，这种好书应该读很多次。

这是一个跟个人比较有关的问题，不同人的不同经历可能对你没有太多参考价值，因为觉得投入产出比对自己非常低，我在当时完全没刷过算法题，现在我也是这么认为的。

不过面向结果编程，为了应对校招来做这件事情完全没有任何问题，应试教育就要有应试教育的对付方法，不是吗？如果精力不够，CSAPP 可以等校招之后再看。

**Comment by vc60er** [comment]
👍，从来没有见过这么牛年轻人，而且还很低调。

**Comment by linghugoogle** [comment]
🐂

**Comment by zhuhaochuan** [comment]
大佬牛逼 文章质量很高，我才硕士毕业发现大佬和我是同一届的，叹服！

**Comment by zhuhaochuan** [comment]
> @Cherrison
> 大佬有读完CSAPP嘛 花费大量时间读完这个值吗, 现在有点纠结, 面试貌似也不会深入问这些.
> 花费那么多时间读CSAPP, 做它实验,,, 最后面试还不如那些, 刷1-200算法题的.
> 春招面试没刷算法题好伤啊
> 

CSAPP是本神书！

**Comment by KylinMountain** [comment]
看了大佬的这篇总结，大佬牛逼，工作五年的我，默默留下了眼泪。
工作后，读书的时间不多了，每天读书就是上下班的地铁上，心累。
现在准备奋发图强，都到六月了，今年才读了六本，心塞。
准备向大佬学习先列个读书清单

**Comment by jash16** [comment]
工作七年，感觉还不如你半年。

**Comment by yanjinbin** [comment]
虽然恬不知耻的加了博主微信，还是不敢问何方高人::>_<:: ？思考深度太牛啦。上次学到的就是trello进行管理还是起了丁点作用的。

**Comment by Ouyangan** [comment]
> @jash16
> 工作七年，感觉还不如你半年。

楼主一直在时刻提醒我,我是个废物

**Comment by FlightSong** [comment]
作者的GO专栏对于go进阶来说提升很大，直接撸源码，讲原理，收益颇多

**Comment by hallestar** [comment]
> @Ouyangan
> > @jash16
> > 工作七年，感觉还不如你半年。
> 
> 楼主一直在时刻提醒我,我是个废物
> 
> 
我也留下了没有技术, 没有积累的泪水😭

**Comment by hefengxian** [comment]
> @jash16
> 工作七年，感觉还不如你半年。

深表同意

**Comment by Iunyasha** [comment]
挖坟，准备开个博客，向博主学习

**Comment by deusyu** [comment]
“别人家的刚毕业”
真的太牛了，顺着「浅谈数据库并发控制 - 锁和 MVCC」爬来的。
看了不少为什么系列，老哥真的是太强啦，还买了本Go语言设计与实现。
